packages/liveblocks-core/src/room.ts

```diff
@@ -2489,7 +2489,10 @@ export function createRoom<
   TUserMeta extends BaseUserMeta,
   TRoomEvent extends Json
 >(
-  options: RoomInitializers<TPresence, TStorage>,
+  options: Omit<
+    RoomInitializers<TPresence, TStorage>,
+    "shouldInitiallyConnect"
+  >,
```

Comment:
Unnecessary for the change, but strictens the type param a bit furhter. This function will ignore the `shouldInitiallyConnect` flag if you pass it.